web-client/src/models/requests/index.ts

```diff
@@ -24,7 +24,13 @@ export enum RequestStatus {
   removed = 'removed',
 }
 
+export type RequestRefType = firebase.firestore.DocumentReference<
+  firebase.firestore.DocumentData
+>;
+
 export interface IRequest extends firebase.firestore.DocumentData {
```

Comment:
requestRef and requestd are used when querying timeline and offers.  Both aren't needed, I put in two for convenience.